src/protocols/two_party_ecdsa/lindell_2017/party_one.rs

```diff
@@ -481,6 +482,7 @@ impl Signature {
         r = r.scalar_mul(&ephemeral_local_share.secret_share.get_element());
 
         let rx = r.x_coor().unwrap().mod_floor(&FE::q());
+        let ry = r.y_coor().unwrap().mod_floor(&FE::q());
```

Comment:
I'm afraid that this trick will not work with other curves. Can you test it when using curv with different elliptic curves? ones where `r.y_coor()` is defined.  If this trick cannot be generalize than I suggest to either move it to KMS / to have to signing functions, one with recid_support and a regular one without